GitUI/BranchTreePanel/BaseBranchLeafNode.cs

```diff
@@ -51,5 +55,28 @@ public override void ApplyStyle()
                 TreeViewNode.ToolTipText = string.Format(TranslatedStrings.ContainedInCurrentCommit, Name);
             }
         }
+
+        public void UpdateAheadBehind(string aheadBehindData, string relatedBranch)
+        {
+            AheadBehind = aheadBehindData;
+            RelatedBranch = relatedBranch;
+        }
+
+        protected override string DisplayText()
+        {
+            return string.IsNullOrEmpty(AheadBehind) ? Name : $"{Name} ({AheadBehind})";
+        }
+
+        protected override void SelectRevision()
+        {
+            TreeViewNode.TreeView?.BeginInvoke(new Action(() =>
```

Comment:
It should be possible to write it as:
```suggestion
            TreeViewNode.TreeView?.BeginInvoke(() =>
```

Reply:
Will change
This was just moved from BaseBranchNode.cs
I assume this goes into the "refactoring - no changes" commit